Common.Packages.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="StyleCop.Analyzers" Version="[1.2.0-beta.435,1.2.0]" />
```

Comment:
Why not use `GlobalPackageReference`? AFAIK this analyzer is used in all projects, no?

Reply:
Everything will import `Common.props` anyways so I just kept them aligned. Actually I can change that and the `Common.Packages.props` file can be removed.

Reply:
I like the idea of removing the `Common.Packages.props`.

Reply:
I've removed it. It also helped getting rid of the error in CI for the demo project.